gettsim/data/abgelt_st.yaml

```diff
@@ -1,14 +1,11 @@
-abgst:
+abgeltungssteuer_satz:
```

Comment:
The file is named abgelt_st. We should use that abbreviation also for the parameters.